app/views/layouts/_footer.html.erb

```diff
@@ -3,7 +3,7 @@
               about_main_index_path,
               remote: true %>
 
-  <% if @current_role.admin? or @current_role.ta? %>
+  <% if @current_role && ( @current_role.admin? or @current_role.ta? ) %>
```

Comment:
I think we can just display this for everyone now. The issues list is a public site. 
I would just remove the if statement entirely so that this issues span is displayed for everyone.